manifest_staging/charts/aad-pod-identity/templates/nmi-podmonitor.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.podMonitor.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
```

Comment:
Could you elaborate more on why pod monitor needs to be part of the pod identity helm charts? IMO this is not something we should package in this helm chart but I'm open to be convinced.